tilelang/layout/swizzle.py

```
@@ -109,19 +109,22 @@ def make_wgmma_swizzled_layout(buffer: BufferLikeType, continuity: int = None, k
 
 # for TCGEN05MMA Intrinsics
 def make_tcgen05mma_swizzled_layout(buffer: BufferLikeType, continuity: int = None, k_major: bool = True):
```

Ruff (0.15.4)

[warning] Unused function argument: `continuity`

(ARG001)